### PR TITLE
Added the Fileserver logic

### PR DESCRIPTION
**Pull request by FGRCL**

### CODE REVIEW

File: src/server/FileServer.java [hunk 1]
```diff
@@ -1,15 +1,89 @@
...
+			response = new HttpResponse(HttpVersion.OnePointOh, Status.OK, body);
+		} catch (FileNotFoundException e) {
+			return new HttpResponse(HttpVersion.OnePointOh, Status.NOT_FOUND, e.getMessage());
+		} catch (FileOutsideDirectoryException e) {
+			return new HttpResponse(HttpVersion.OnePointOh, Status.FORBIDDEN, e.getMessage());
+		}
+		return response;
+	}
+	
+	private HttpResponse getDirectotyContent(URI uri) {
```
Comment (felixlapierre) [inline src/server/FileServer.java]: i love to store my files in a directoty

File: src/server/library/HttpResponse.java [hunk 2]
```diff
@@ -14,6 +14,13 @@ public HttpResponse(HttpVersion version, Status status) {
         this.version = version;
         this.headers = new LinkedList<>();
     }
+    
+    public HttpResponse(HttpVersion version, Status status, String body) {
+        this.status = status;
+        this.version = version;
+        this.headers = new LinkedList<>();
+        this.body = body;//dad bods are hot
```
Comment (felixlapierre) [inline src/server/library/HttpResponse.java:22]: are they tho

File: text.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+Hello word
```
Comment (felixlapierre) [inline text.txt:1]: couldn't afford the third L?

File: src/server/FileServer.java (hunk 1 above)
Comment (FGRCL) [reply src/server/FileServer.java]: that's how they call those in Linux

File: src/server/library/HttpResponse.java (hunk 2 above)
Comment (FGRCL) [reply src/server/library/HttpResponse.java:22]: YES

File: text.txt (hunk 3 above)
Comment (FGRCL) [reply text.txt:1]: I being paid per character

Comment (FGRCL) [reply text.txt:1]: but that file should be deleted after the test

**Pull request merged**